### PR TITLE
document.execCommand(), based on MDN and testing on CBT

### PR DESCRIPTION
**Pull request by dandv**
Based on https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand and manual testing at crossbrowsertesting.com

### CODE REVIEW

**Comment by cvrebert** [comment]
@dandv Travis JSON validation check is failing; you need to add https://github.com/Fyrd/caniuse/blob/ca56a2a7cc20ecdfb6a53fe377224e4a75f264ae/sample-data.json#L223

File: features-json/document-exec-command.json [hunk 1]
```diff
@@ -0,0 +1,224 @@
+{
+  "title":"Document.execCommand()",
+  "description":"Allows running commands to manipulate the contents of an editable region in a document switched to designMode",
+  "spec":"https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html#execcommand()",
+  "status":"wd",
```
Comment (cvrebert) [inline features-json/document-exec-command.json]: It's an Editor's Draft, not a Working Draft

File: features-json/document-exec-command.json [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+    },
+    "ie_mob":{
+      "10":"u",
+      "11":"u"
+    },
+    "and_uc":{
+      "9.9":"u"
+    }
+  },
+  "notes":"To determine what commands are supported, see Document.QueryCommandSupported()",
```
Comment (cvrebert) [inline features-json/document-exec-command.json]: The Q should be lowercase

File: features-json/document-exec-command.json (hunk 1 above)
Comment (dandv) [inline features-json/document-exec-command.json]: What's the code for Editor's Draft at https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md#supported-changes ?

**Comment by Fyrd** [comment]
Thanks!

**Pull request merged**